CoffeeChat/Cells/OnboardingTableViewCell.swift

```diff
@@ -63,23 +68,48 @@ class InterestsTableViewCell: UITableViewCell {
 
         titleLabel.snp.makeConstraints { make in
             make.leading.equalTo(interestImageView.snp.trailing).offset(textSidePadding)
-            make.top.equalTo(cellBackgroundView).inset(sidePadding)
+            if showingInterests {
+                make.top.equalTo(cellBackgroundView).inset(sidePadding)
+            } else {
+                make.centerY.equalToSuperview()
+            }
         }
 
-        categoriesLabel.snp.makeConstraints { make in
-            make.leading.equalTo(titleLabel)
-            make.top.equalTo(titleLabel.snp.bottom)
+        if showingInterests {
+            categoriesLabel.snp.makeConstraints { make in
+                make.leading.equalTo(titleLabel)
+                make.top.equalTo(titleLabel.snp.bottom)
+            }
         }
     }
 
     func configure(with interest: Interest) {
         titleLabel.text = interest.name
         categoriesLabel.text = interest.categories
+        // Determine if a relayout is needed (was showing Group and configured with an Interest)
+        if !initialized || !showingInterests {
+            showingInterests = true
```

Comment:
you can do `showingInterests.toggle()`

Reply:
the toggle may not work based the default state of `showingInterests` if the cell in uninitialized, but I moved the showingInterests logic into `relayoutSubviews`